.github/workflows/build-and-test.yml

```diff
@@ -84,7 +84,7 @@ jobs:
       - name: Install OpenMPI
         run: |
           sudo apt-get -yy update
-          sudo apt-get install -y libopenmpi-dev
+          sudo apt-get install -y libopenmpi-dev cmake libboost-all-dev libeigen3-dev pkg-config
```

Comment:
If you don't have to install preCICE (see below), you should be able to get away without these dependencies.

Reply:
I am not installing preCICE completely, just running the cmake command, hence I need these dependencies.

Reply:
Understood :+1: